pinot-controller/src/main/java/com/linkedin/pinot/controller/validation/StorageQuotaChecker.java

```diff
@@ -63,12 +63,22 @@ public StorageQuotaChecker(AbstractTableConfig tableConfig, TableSizeReader tabl
   public QuotaCheckerResponse isSegmentStorageWithinQuota(@Nonnull File segmentFile, @Nonnull String tableNameWithType,
       @Nonnull String segmentName,
       @Nonnegative int timeoutMsec) {
-    Preconditions.checkNotNull(segmentFile);
-    Preconditions.checkNotNull(tableNameWithType);
-    Preconditions.checkNotNull(segmentName);
-    Preconditions.checkArgument(timeoutMsec > 0, "Timeout value must be > 0, input: %s", timeoutMsec);
-    Preconditions.checkArgument(segmentFile.exists(), "Segment file: %s does not exist", segmentFile);
-    Preconditions.checkArgument(segmentFile.isDirectory(), "Segment file: %s is not a directory", segmentFile);
+    try {
+      Preconditions.checkNotNull(segmentFile);
```

Comment:
Okay, so for user errors, I should continue to allow the api to throw an exception? My thought process was that for user errors, we should always log infos; is that incorrect? I can remove the try catch for the null preconditions, but should I leave them for illegal argument exceptions?